docs/governance/templates/release_issue.md

```diff
@@ -33,7 +33,7 @@ and copy it into a release issue. Fill in relevant values, found inside {}
 - [ ] `git checkout main && git pull --rebase upstream main`
 - [ ] Run `make release-deploy-site`
       - For example, if you are creating the {version} release, then this would deploy the {version}-1 service (release minus one, and then replace dots with dashes)
-- [ ] Run `make build-release` to ensure all example images exist on us-docker.pkg.dev/agones-images/examples.
+- [ ] Run `make build-release` to ensure all example images exist on us-docker.pkg.dev/agones-images/examples and to store the artifacts in cloud storage.
```

Comment:
This looks great! The only other thought I had to add, is to mention when to download the files form cloud storage so they can be attached to the Github release. 👍🏻

Reply:
Updated artifacts storage related steps in release checklist